python/taichi/lang/impl.py

```diff
@@ -138,6 +139,26 @@ def chain_compare(comparators, ops):
     return ret
 
 
+@taichi_scope
+def func_call(func, *args, **kwargs):
```

Comment:
1. Maybe a more specific name?
2. Does this need to be placed under `ti` for `ASTTransformer_Checks` to work?

Basically, i'm trying to limit the scope of this func, since it's just for internal usage?

Reply:
Not really, we may have more hack here. If I name this `ti.func_call_check` then it would be redulant to do some hack other than check, e.g.: auto transform a non-func into `@ti.func` in invocation.